sample/README.md

```diff
@@ -4,7 +4,7 @@ Sample application for the Urban Airship SDK.
 
 ## Setup
 
-1) Update the [airshipconfig.properties](src/main/assets/airshipconfig.properties) file
+1) Update the [airshipconfig.properties](src/main/assets/airshipconfig.properties.sample) file
```

Comment:
@rlepinski Was going to get to this today, but seems you beat me to it. 😄 

Managed to figure out that nuance when I got my app fully up and running a few hours ago. Thanks! 👌